### PR TITLE
Constants: Import appdirs when needed

### PR DESCRIPTION
**Pull request by sils1297**
Fixes https://github.com/coala-analyzer/coala/issues/1957

### CODE REVIEW

**Comment by AbdealiJK** [comment]
It's also probably a better idea to read VERSION in `coalib/__init__.py` and use `coalib.__version__` in Constants.
This was version is isolated from other constants, which is actually the core problem.

**Comment by sils1297** [comment]
agreed.

File: coalib/output/dbus/BuildDbusService.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from distutils.core import Command
 from distutils.errors import DistutilsOptionError
 
-from coalib.misc import Constants
+from coalib import BUS_NAME
```
Comment (Adrianzatreanu) [inline coalib/output/dbus/BuildDbusService.py]: shouldn't this be ``from coalib.__init__ import BUS_NAME`` ? or it works like this too?

**Comment by sebix** [comment]
Yes, works.

File: coalib/output/dbus/BuildDbusService.py (hunk 1 above)
Comment (sils1297) [inline coalib/output/dbus/BuildDbusService.py]: that's who you use the `__init__` so people can import from the "module"

**Comment by Adrianzatreanu** [comment]
ack 906e869

**Comment by Adrianzatreanu** [comment]
ce6061d is ready

**Comment by Adrianzatreanu** [comment]
ack ce6061d

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**